playbooks/openstack-update_and_reboot_caasp.yml

```diff
@@ -63,5 +63,5 @@
         validate_certs: no
       register: result
       until: result.status == 200
-      retries: 60
+      retries: 100
```

Comment:
nit: probably shouldn't be on the same commit

Reply:
removed